packages/now-cli/src/commands/env/add.ts

```diff
@@ -127,7 +204,16 @@ export default async function add(
   const addStamp = stamp();
   try {
     await withSpinner('Saving', () =>
-      addEnvRecord(output, client, project.id, envName, envValue, envTargets)
+      addEnvRecord(
+        output,
+        client,
+        project.id,
+        // @ts-ignore
```

Comment:
Why the ts-ignore?

Reply:
Removed in https://github.com/vercel/vercel/pull/5372/commits/3237f0fff7c5b1de0aef5268e5f7e95fdcfe37c7 👍